test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -1822,11 +1822,19 @@ def convert_to_dictlist(self, l, attribs=None, mergelines=True, id=None):
 
         if mergelines:
             lines = []
+            count = 1
             for i in range(len(l)):
                 if l[i].startswith('\t'):
                     _e = len(lines) - 1
                     lines[_e] = lines[_e].strip('\r\n\t') + \
                         l[i].strip('\r\n\t')
+                elif not l[i].startswith(' ') and l[i-count].startswith('\t'):
```

Comment:
`l[i-count]` this can go wrong is `i` is 0 as the `count` is starting from 1, aka if `i` is 0 and `count` is 1 then it will check the last line instead the previous line.